### DISCUSSION TITLE
2.1 inch Turing, need help!

### DISCUSSION THREAD

**Discussion by fern4nd0sb** [mathoudebine/turing-smart-screen-python#440] [General]
I just received it but seems it came with some issues in ROM, I can't apply any theme, is there a way to fix it? I can't not even official forum helps
![image (2)](https://github.com/mathoudebine/turing-smart-screen-python/assets/81946887/560c61c7-0e34-42f6-9618-ede49a7346f9)
![image (3)](https://github.com/mathoudebine/turing-smart-screen-python/assets/81946887/4950f486-8d56-44fb-9f41-e0ae2ff49e4b)

**Comment by mathoudebine** [comment]
Hello, maybe your screen model is defective. Did you make it work before?
I see you have no internal memory from your screen capture:
![image](https://github.com/mathoudebine/turing-smart-screen-python/assets/38615348/3d9fd604-ebbe-4313-962a-21674b13aaa9)
This is certainly why you can't apply any theme or upload any video.
If you bought it from a reseller, you can contact them to have it replaced
